common/src/main/java/tds/common/configuration/RestTemplateConfiguration.java

```diff
@@ -1,27 +1,35 @@
 package tds.common.configuration;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 import org.springframework.http.client.BufferingClientHttpRequestFactory;
 import org.springframework.http.client.SimpleClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
+import tds.common.web.interceptors.RestTemplateEventLoggingInterceptor;
 import tds.common.web.interceptors.RestTemplateLoggingInterceptor;
 
 @Configuration
 @Import(JacksonObjectMapperConfiguration.class)
 public class RestTemplateConfiguration {
 
+    @Autowired
```

Comment:
Instead of autowiring this field can you see if it can just be added to the restTemplate() method as an argument?  Since this is a configuration class Spring should inject it.